app/javascript/components/patient/laboratory/FirstPositiveLaboratory.js

```diff
@@ -105,6 +112,7 @@ class FirstPositiveLaboratory extends React.Component {
 
 FirstPositiveLaboratory.propTypes = {
   lab: PropTypes.object,
+  disableDelete: PropTypes.bool,
```

Comment:
thoughts on calling this `preventDelete` or reversing it and calling it `allowDelete`.  The name is clear, just don't love it

Reply:
[9d0764e](https://github.com/SaraAlert/SaraAlert/pull/1205/commits/9d0764e18fcec02f59b0d8ce888ddf4e029a0e36)